pkg/mutation/match/match.go

```diff
@@ -22,7 +23,14 @@ const Wildcard = "*"
 // Match selects objects to apply mutations to.
 // +kubebuilder:object:generate=true
 type Match struct {
-	Kinds []Kinds `json:"kinds,omitempty"`
+	// Source determines if the gatekeeper config should be used to expand generator
```

Comment:
This does not gate expansion... it gates whether the policy/mutator should be applied to expanded resources:

"Source determines whether generated or original resources are matched. Accepts ..."